smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/GetTogglesIT.java

```diff
@@ -62,6 +69,22 @@ public void testTogglesEndpointReturnsStaticEnabledFlagInJsonResponseOnPublish()
         sharedTestResponseAlwaysContainsEnabledFlag(adminPublish);
     }
 
+    /**
+     * Validate format of AEM version containing state qualifier using six digits
+     */
+    @Test
+    public void testAboutPageVersionFormatWithToggleQualifier() throws ClientException, IOException, SAXException, ParserConfigurationException {
+        SlingHttpResponse response = adminAuthor.doGet("mnt/overlay/granite/ui/content/shell/about.html", 200);
+        final String regex = "^Adobe Experience Manager [\\d]{4}.[\\d]{2}.[\\d]+.[\\d]{8}T[\\d]{6}Z-[\\d]{6}$";
+
+        Document doc = DocumentBuilderFactory.newInstance()
+            .newDocumentBuilder()
+            .parse(new InputSource(new StringReader(response.getContent())));
+
+        String aemVersionLine = doc.getElementsByTagName("p").item(0).getTextContent().trim();
```

Comment:
This seems brittle and could be easily broken if another paragraph is inserted before the version or if the `<p>` tag is changed to something else. Wouldn't be better to match the regex against the whole page?

Reply:
I wanted to avoid adding the `</p>` as part of the regex check, to ensure that the element contains the full version value. Have updated to check in any `<p>` tag.

Will iterate to check any element's text. Usually we would add a `data-test-` attribute to that element, to make sure we target the exact placeholder structure for the version holder.